.github/workflows/tests.yaml

```diff
@@ -27,6 +27,27 @@ jobs:
             dependencies: "highest"
             phpunit-flags: "--fail-on-risky"
 
+    env:
+      MYSQL_HOST: 127.0.0.1
+      MYSQL_PORT: 3306
+      MYSQL_USER: root
+      MYSQL_PASSWORD: pimcore
+      MYSQL_DATABASE: pimcore
+      MYSQL_SERVER_VERSION: "10.11.5-MariaDB"
+
+    services:
+      mariadb:
+        image: mariadb:10.11.5
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ROOT_PASSWORD: pimcore
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=5
```

Comment:
Sieh mal hier, da haben wir das auch schon: https://github.com/teamneusta/pimcore-translation-migration-bundle/blob/main/.github/workflows/tests.yaml

Bzw. auch in der Quality Bundle Base, die ich hier aber nicht verlinken kann.

Reply:
typically, if you don't need root, when mariadb is started with all variable like USER, PASSWORD, DATABASE. The user should have all privilidges for the database. And that includes deleting his own database.

Reply:
I just tried again with a non-root user and it fails with:
> RuntimeException: Error running "doctrine:database:drop":
> In ExceptionConverter.php line 101:
> 
> An exception occurred in the driver: SQLSTATE[HY000] [1045] Access denied for user 'pimcore'@'172.18.0.1' (using password: YES)

Reply:
Ah, maybe this is because of the different IP. We normally use localhost or 127.0.0.1. But in this case the IP is different because GitHub Actions works different. So than you must use the root user.